scripts/linters/pylint_extensions.py

```diff
@@ -1484,31 +1484,63 @@ class RestrictedImportChecker(checkers.BaseChecker):
             'domain layer and controller layer respectively.'),
     }
 
-    def visit_import(self, node):
-        """Visits every import statement in the file.
+    options = (
+        (
+            'forbidden-imports',
+            {
+                'default': ('core.storage:core.domain',),
+                'type': 'csv',
+                'metavar': '<comma separated list>',
+                'help': 'List of disallowed imports.'
+            }
+        ),
+    )
+
+    def iterate_forbidden_imports(self, node):
+        """Yields pairs of module name and forbidden imports.
 
         Args:
             node: astroid.node_classes.Import. Node for a import statement
                 in the AST.
+
+        Yields:
+            tuple(str, str). Yields pair of module name and forbidden import.
         """
+        splited_module_to_forbidden_imports = [
+            forbidden_import.strip().split(':')
+            for forbidden_import in self.config.forbidden_imports
+        ]
+        module_to_forbidden_imports = list(
+            (
+                'oppia.%s' % forbidden_import[0].strip(),
```

Comment:
why do we need oppia here? (Can we pass it through the config (pylintrc) itself?)

Reply:
I wanted to keep the both parts of the module and forbidden imports the same. But on the other hand, this then means that the extension is then oppia specific. I'm fine with both solutins.

Reply:
I moved the oppia to the .pylintrc and added an explanation of this to the help.